contracts/AedexV2Pair.aes

```diff
@@ -68,76 +71,55 @@ contract AedexV2Pair =
         , min_liquidity: int
         , debug_time: option(int)
         ) = {
-              total_supply = 0
-            , balance_of   = {}
-            , locked_liquidity = 0
-            , allowance    = {}
-            , factory = factory
-            , token0 = token0
-            , token1 = token1
-            , reserve0 = 0
-            , reserve1 = 0
-            , block_timestamp_last = 0
+              total_supply           = 0
+            , balance_of             = {}
+            , locked_liquidity       = 0
+            , allowances             = {}
+            , factory                = factory
+            , token0                 = token0
+            , token1                 = token1
+            , reserve0               = 0
+            , reserve1               = 0
+            , block_timestamp_last   = 0
             , price0_cumulative_last = 0
             , price1_cumulative_last = 0
-            , k_last = 0
-            , debug_time = debug_time
-            , min_liquidity = min_liquidity
+            , k_last                 = 0
+            , debug_time             = debug_time
+            , min_liquidity          = min_liquidity
         }
 
-    // INTERNALS
-
-    function get_balances() =
-        let balance0 = Option.default(0,state.token0.balance(Contract.address))
-        let balance1 = Option.default(0,state.token1.balance(Contract.address))
-        (balance0,balance1)
-
-
-    function timestamp() = Option.default(Chain.timestamp,state.debug_time)
-
-    stateful entrypoint set_debug_time(time: int) =
-        require(Option.is_some(state.debug_time),"AedexV2Pair: NOT_DEBUG_MODE")
-        put( state{ debug_time= Some(time)  } )
-
-    // update reserves and, on the first call per block, price accumulators
-    stateful function update(balance0: int, balance1: int, reserve0: int,reserve1: int  ) =
-        require( balance0 >= 0 && balance1 >= 0 && reserve0 >= 0 && reserve1 >= 0,"AedexV2Pair: LP" )
+    //------------------------------------------------------------------------------
+    // VIEWERS
+    //------------------------------------------------------------------------------
+    //TODO: we might want as meta_info to reflect token1&token0 meta_info
+    //find a suitable combination
+    entrypoint meta_info(): IAEX9Minimal.meta_info = {
+          name = "Aedex v2",
+          symbol = "AEDX-V2",
+          decimals = 18
+        }
 
-        let block_timestamp = timestamp()
+    //TODO: even if we have mint and burn, the entrypoints signature and business
```

Comment:
I would say to officially not support mintable and burnable then as standardized, so a third party application won't be confused